### DISCUSSION TITLE
Is z ONE executable?

### DISCUSSION THREAD

**Discussion by halloleo** [ajeetdsouza/zoxide#675] [Q&A] [answered]
Is `z` _one_ executable which I could download standalone for amd64? 

(I am asking because I want o install it on a machine without internet connection.)

**Comment by ajeetdsouza** [comment, marked as answer]
`zoxide` is the executable you're looking for. You can download it from the [Releases](https://github.com/ajeetdsouza/zoxide/releases) page.

Once you set it up on your shell (instructions in the README), it initializes multiple commands.
